ConorMFinn/CO600-SeriousCyberSec
# Changed the colour of the carpet tile pattern

**Description of work.**

The colour of the pattern on the carpet tiles was clashing with the rest of the scene so I've changed it to be a softer pattern.

**To test:**

Start the game and check the floor tile doesn't clash with the rest of the scene.

Fixes #53
